src/handbook/marketing/blog.md

```diff
@@ -25,7 +25,35 @@ Content creation and blogging are effective ways to communicate with our communi
 1. When both reviews are done*, the Author or SEO reviewer can merge. In the event of an external author, SEO/content review will merge.
 1. Once merged, SEO Reviewer will promote on FlowFuse Social channels. Check the [social media planning and posting section](/handbook/marketing/social-media/#planning-and-posting) for guidance and considerations.
 
-*It's ok to merge without a blog tile, as long as there's an art request open with a clear deadline and it doesn't get shared on socials without having the image.
+### Temporary Blog Tile Image Process
+
+**When no designer is available and the content is time-sensitive**, you may create a temporary blog tile so we can publish without delay. Please still open an art request so a designer can update the image later.
+
+Options and steps:
+
+1. **Default Image**  
```

Comment:
Why do you use bold everywhere? Overuse makes it feel like everything is important and so nothing is important